ModTek/ProgressPanel.cs

```diff
@@ -56,8 +56,29 @@ private IEnumerator RunWorkList()
                 foreach (var workFunc in WorkList)
                 {
                     var workEnumerator = workFunc.Invoke();
-                    while (workEnumerator.MoveNext())
+                    bool didWork;
+
+                    do
                     {
+                        try
+                        {
+                            didWork = workEnumerator.MoveNext();
+                        }
+                        catch (Exception e)
+                        {
+                            Logger.FlushLogStream();
+                            Logger.Log($"");
+                            Logger.Log($"Uncaught ModTek exception!");
+                            Logger.Log(e.ToString());
+                            Logger.FlushLogStream();
+
+                            Slider.value = 1.0f;
+                            SliderText.text = "ModTek Died!";
+                            LoadingText.text = "See Mods/.modtek/ModTek.log";
```

Comment:
"\\" instead of "/" in case you are on windows? oh what about dynamically getting this path for the platform?